contributors/devel/welcome-to-kubernetes-new-developer-guide.md

```diff
@@ -0,0 +1,128 @@
+
+Welcome to Kubernetes! (New Developer Guide)
+============================================
+
+2017-06-21 through 2017-08-06
+Original author: [Alexander Campbell](https://github.com/alexandercampbell)
+
+---
+
+_This document assumes that you know what Kubernetes does. If you don't, search
+YouTube for ["Kubernetes"](https://www.youtube.com/watch?v=8SvQqZNP6uo) and try
+the demo at [https://k8s.io/](https://k8s.io/)._
+
+Introduction
+------------
+
+Have you ever wanted to contribute to the coolest cloud technology? This
+document will help you understand the organization of the Kubernetes project and
+direct you to the best places to get started. By the end of this doc, you'll be
+able to pick up issues, write code to fix them, and get your work reviewed and
+merged. If you have questions that aren't answered in this document, feel free
+to suggest changes to it directly or email me at
+[alexanderhcampbell@gmail.com](mailto:alexanderhcampbell@gmail.com).
```

Comment:
I decided to remove this entire sentence since the information is duplicated in a later paragraph

```markdown
There is an entire SIG
([sig-contributor-experience](https://github.com/kubernetes/kubernetes/wiki/CLA-FAQ))
devoted to improving your experience as a contributor. Contributing to
Kubernetes should be easy. If you find a rough edge, let us know!  File an issue
in the appropriate repository if you know it, send a message on [our Slack
channel](https://kubernetes.slack.com/messages/C1TU9EB9S/details) or to the
[owners](https://github.com/kubernetes/community/blob/master/contributors/devel/owners.md)
of contributor-experience. Better yet, help us fix it by joining the SIG; just
show up to one of the [bi-weekly
meetings](https://docs.google.com/document/d/1qf-02B7EOrItQgwXFxgqZ5qjW0mtfu5qkYIF1Hl4ZLI/edit).
```